scripts/model_gen_ng/model_gen_ng/templates/struct_late.tpl

```diff
@@ -0,0 +1,79 @@
+#pragma once
+
```

Comment:
* [ ] Copyright notice
* [ ] All generated files should clearly state they are generated. I can see people getting frustrated changing them.

Reply:
Very good point, added copyright notice and "this is an autogenerated file" notice